source/_integrations/recorder.markdown

```diff
@@ -12,7 +12,7 @@ ha_codeowners:
 ha_integration_type: system
 ---
 
-The `recorder` integration is responsible for storing details in a database, which then are handled by the [`history`](/integrations/history/) integration.
+The `recorder` integration is responsible for storing details in a database, which then are handled by the [`history`](/integrations/history/) integration. This integration is by default enabled as dependency of the [`history`](/integrations/history/) integration.
```

Comment:
I don't think that we need to link history twice. Maybe just mention "This integration is enabled by default."

Reply:
I think the whole relation with the history integration can be removed from this part.

Reply:
> I think the whole relation with the history integration can be removed from this part.

I agree, removed it.